data/quickstarts/pachyderm-beginner-tutorial-notebook.yaml

```
@@ -0,0 +1,51 @@
+kind: ConsoleQuickStart
+metadata:
+  name: pachyderm-beginner-tutorial-notebook
+  annotations:
+    opendatahub.io/categories: 'AI/Machine learning,Getting started,Jupyter notebook,Pipeline,Python'
+
+spec:
+  displayName: Pachyderm Beginner Tutorial Notebook
+  appName: pachyderm
+  durationMinutes: 30
+  icon: 'images/pachyderm.svg'
+  description: This quick start will help you load Pachyderm beginner tutorial notebook,
+   a notebook introduction to Pachyderm, using the pachctl command line utility to illustrate the basics of Pachyderm data repositories and pipelines
```

```suggestion
Load a notebook introduction to Pachyderm using the pachctl command line utility to illustrate the basics of Pachyderm data repositories and pipelines.
``` 

This would be the card description.

---

@nadegepepin Could you please review?